PENDING_RELEASE_NOTES.md

```diff
@@ -2,11 +2,16 @@
 ## Version 17.4.0
 _Date TBD_
 
-### View/Table modifications
+### Table modifications
 * Create table 'filter' to handle storing user's filters
-* Create table 'filter_type' to indicate the type of filter 
+* Create table 'filter_type' to indicate the type of filter
+
+### View Modifications
+* Updated "acb_is_deleted" field in certified_product_details view to be "acb_is_retired".
```

Comment:
Although git will let you merge this file, this line and the one below were already deployed in the previous release. Check existing release notes when merging and update accordingly.